src/app/features/Auth/forget-password/forget-password.component.html

```
@@ -1,80 +1,274 @@
+          class="w-full flex-1 flex flex-col"
+          (submit)="enterEmail(email.value)"
+        >
+          <div class="mb-4 flex-1">
+            <label
+              for="email-address-icon"
+              class="block text-sm font-medium text-gray-700 mb-1.5"
+              >Email</label
+            >
+            <p-iftalabel class="w-full block">              <input
+                #email
+                [ngClass]="{ 'ng-dirty ng-invalid': emailInvalid }"
```

HTMLHint (1.5.0)

[error] The attribute name of [ [ngClass] ] must be in lowercase.

(attr-lowercase)